src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetSchedulingAndDescription/index.tsx

```diff
@@ -0,0 +1,264 @@
+
+  const descriptionMutation = useUpdateVideo(video.id, {
+    onSuccess: () => {
+      toast.success(intl.formatMessage(messages.updateVideoSuccess));
+    },
+    onError: (err) => {
+      report(err);
+      toast.error(intl.formatMessage(messages.updateVideoFail));
+      setDescriptionState(video.description);
+    },
+  });
+
+  const timeoutId = useRef<Maybe<number>>();
+
+  const debounce = (
+    fn: (updatedVideoProperty: Partial<Video>) => void,
+    ms = 500,
+  ) => {
+    return (updatedVideoProperty: Partial<Video>) => {
+      window.clearTimeout(timeoutId.current);
+      timeoutId.current = window.setTimeout(() => fn(updatedVideoProperty), ms);
+    };
+  };
+
+  const debouncedUpdatedVideo = React.useRef(
+    debounce((updatedVideoProperty: Partial<Video>) => {
+      if (updatedVideoProperty.description === '') {
+        toast.error(intl.formatMessage(messages.updateDescriptionBlank));
+        setDescriptionState(video.description);
+        return;
+      }
+      descriptionMutation.mutate(updatedVideoProperty);
+    }),
+  ).current;
+
+  const handleChange = async (updatedVideoProperty: Partial<Video>) => {
+    window.clearTimeout(timeoutId.current);
+    debouncedUpdatedVideo(updatedVideoProperty);
+  };
+
+  return (
+    <DashboardVideoLiveWidgetTemplate
+      infoText={intl.formatMessage(messages.info)}
+      initialOpenValue={true}
+      title={intl.formatMessage(messages.title)}
+    >
+      <Box direction="column" gap="small">
+        <Grommet theme={schedulingFieldsTheme}>
+          <SchedulingFields
+            estimatedDuration={video.estimated_duration}
+            disabled={video.live_state !== liveState.IDLE}
+            margin="none"
+            onEstimatedDurationChange={(estimatedDuration) => {
+              handleChange({
+                estimated_duration: estimatedDuration,
+              });
+            }}
+            onStartingAtChange={(startingAt) => {
+              handleChange({
+                starting_at: startingAt,
+              });
+            }}
+            startingAt={video.starting_at}
+          />
+        </Grommet>
+
+        <Box
+          border={{
+            color: '#81ade6',
+            size: 'xsmall',
+            style: 'dashed',
+          }}
+          direction="row"
+          justify="between"
+          pad={{ horizontal: 'medium', vertical: '12px' }}
+          round="6px"
+        >
+          {video.starting_at ? (
+            <Fragment>
+              <StyledText color="blue-active" size="0.875rem">
+                {intl.formatMessage(messages.endDateWebinarLabel)}
+              </StyledText>
+              <StyledText color="blue-active" size="0.875rem">
+                {video.estimated_duration
+                  ? DateTime.fromISO(video.starting_at)
+                      .plus(Duration.fromISOTime(video.estimated_duration))
+                      .setLocale(intl.locale)
+                      .toLocaleString(DateTime.DATETIME_SHORT)
```

Comment:
displaying in english the date with this format `mm/dd/yyyy` is not what we want. This is the rule we want:

- in french: `dd/mm/yyyy`
- otherwise: `yyyy/mm/dd`

Reply:
done